### PR TITLE
add proxy for salvo dependency install doc

### PR DESCRIPTION
**Pull request by xu1zhou**
Executing install_deps.sh with sudo may lost proxy environment and cause network fail. Add doc example to avoid that.

### CODE REVIEW

File: salvo/README.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ workflow documentation: [ENVOY_DEVELOP_WORKFLOW.md](./ENVOY_DEVELOP_WORKFLOW.md)
 
 The [`install_deps.sh`](./install_deps.sh) script can be used to install any dependencies required
 by Salvo.
+Dependencies installation needs root priviliges to perform. Using below command to make sure proxy(if any) works fine with root context.
+```bash
+sudo http_proxy="example.com:port" https_proxy="example.com:port" ; ./install_deps.sh
```
Comment (gyohuangxin) [inline salvo/README.md:27]: This command doesn't make sure to use root to run this script, it should be 
```
sudo http_proxy="example.com:port" https_proxy="example.com:port" ./install_deps.sh
```

**Review by gyohuangxin — CHANGES_REQUESTED** [review]

**Comment by gyohuangxin** [comment]
@xu1zhou I think you need to sign your all commits.

**Review by gyohuangxin — DISMISSED** [review]
LGTM, @mum4k  do you have any comment?

File: salvo/README.md [hunk 2]
```diff
@@ -22,6 +22,10 @@ workflow documentation: [ENVOY_DEVELOP_WORKFLOW.md](./ENVOY_DEVELOP_WORKFLOW.md)
 
 The [`install_deps.sh`](./install_deps.sh) script can be used to install any dependencies required
 by Salvo.
+Dependencies installation needs root priviliges to perform. Using below command to make sure proxy(if any) works fine with root context.
```
Comment (mum4k) [inline salvo/README.md:25]: Typo: privileges.

File: salvo/README.md [hunk 3]
```diff
@@ -22,6 +22,10 @@ workflow documentation: [ENVOY_DEVELOP_WORKFLOW.md](./ENVOY_DEVELOP_WORKFLOW.md)
...
+```bash
+sudo http_proxy="example.com:port" https_proxy="example.com:port"  ./install_deps.sh
```
Comment (mum4k) [inline salvo/README.md:27]: Can we also include a command example for running **without** a proxy? That may still be the majority of use cases. Something like:

Dependencies installation needs root privileges to perform. Run the following command to install the required dependencies:

```
sudo ./install_deps.sh
```

If you are using a proxy server, run the following command to download any packages via a proxy server:

```
sudo http_proxy="example.com:port" https_proxy="example.com:port" ./install_deps.sh
```

**Review by mum4k — CHANGES_REQUESTED** [review]

File: salvo/README.md (hunk 3 above)
Comment (xu1zhou) [reply salvo/README.md:31]: gonna take your helpful advice

**Review by mum4k — APPROVED** [review]
Looks great, thank you!

**Review by gyohuangxin — APPROVED** [review]

**Comment by gyohuangxin** [comment]
@mum4k I don't have the merge permission, can you help merge it? Thanks.

**Pull request merged**